app/kumactl/cmd/apply/apply_test.go

```diff
@@ -178,6 +182,40 @@ var _ = Describe("kumactl apply", func() {
 		Expect(resource.Meta.GetNamespace()).To(Equal("default"))
 	})
 
+	It("should apply a new Dataplane resource from URL", func() {
```

Comment:
Let's add a few negative test cases (use `DescribeTable(...)`):
* one with a URL that returns status code `301`
* one with a URL that returns status code `400`
* one with a URL that returns status code `404`
* one with a URL that returns status code `500`

Let's verify the output of `kumactl apply` in that case.

E.g.,
```
// when
err := rootCmd.Execute()
// then
Expect(err.Error()).To(MatchRegexp(requiredFlagNotSet("address")))
// and
Expect(outbuf.String()).To(Equal(`Error: required flag(s) "address" not set`))
// and
Expect(errbuf.Bytes()).To(BeEmpty())
```